### ISSUE TITLE
removeAllTags() in on('dropdown:select') not working?

### ISSUE THREAD

**Issue by Tagirijus** [yairEO/tagify#860]
Hey there,

either I am not getting how to use it or there is missing something? The thing is: I want to use a dropdown (tagify in mode "select") to be able to choose an option which will start a custom javascript function **and clear the field** immediateily afterwards again. I bound a custom JS function to the _'dropdown:select'_ event like this:

```js
var element = document.getElementById('myInputFieldOrSo');

tagify_element = new Tagify(element, {mode: "select"});

tagify_element.on('dropdown:select', function (e) {
	myCustomJSFunction(e);
});

function myCustomJSFunction(e) {
	e.detail.tagify.removeAllTags();
	console.log('Great! Something was selected and the field should be cleared now again!');
}
```

Unfortunately nothing happens. Is it even possible to clear the tagify-input field immediately after the user has chosen something (even if this might be an uncommon way of using tagify, I guess)?

Thanks for your help!

**Comment by yairEO** [comment]
what is `tagify_element`? can you show the rest of the code?

**Comment by Tagirijus** [comment]
Hey, sorry for my fault. I updated the initial post. `tagify_element` is just a tagify object.

**Comment by Tagirijus** [comment]
Oh, I found out another thing (sorry that this is also a bit missing in my initial example and might confuse a bit):

In my personal use case it's about a "mode: 'select'" tagify. I just tested it with a normal tagify and there the behaviour is like this: after selecting from the dropdown, the tagify field does not get cleared, **but** it seems like the tags might be deleted, because: when I select another item, it gets added, while the other one is gone suddenly. A `tagify.update()` doesn't change this.

**Comment by Tagirijus** [comment]
It seems like it does work. I am not 100% sure, but I guess in my individual code the issue was that I tried to remove the tags outside the async fetch() function. Also I changed my code so that all the tagify objects in my code are accessible with a unique ID now. Before that I initialized all tagify objects with a temp-variable, which was used to init the next tagify as well (probably a very noobish thign to do, I know ...).

Thanks for taking your time and sorry for any inconvenience. Also thanks for this unbelievable great script! It is very handy and saves time. (=
